src/Microsoft.DotNet.Build.Tasks/PackageFiles/RunnerTemplate.Unix.txt

```diff
@@ -95,7 +95,15 @@ if [ "$(uname -s)" == "Darwin" ]; then
 elif [ "$(uname -s)" == "Linux" ]; then
   # On Linux, we'll enable core file generation unconditionally, and if a dump
   # is generated, we will print some useful information from it and delete the
-  # dump immediately. 
+  # dump immediately.
+
+  if [ -e /proc/self/coredump_filter ]; then
```

Comment:
This question goes for your earlier core_dump stuff in general https://github.com/adityamandaleeka/buildtools/commit/904d081f00b67ed1b18060b85742a727738f3c99. 

I like the idea of capturing dumps but is there a better place then the runner scripts to do it?